src/routes/safe/components/Transactions/TxList/TxInfoTransfer.tsx

```diff
@@ -14,21 +15,27 @@ export const TxInfoTransfer = ({ txInfo }: { txInfo: Transfer }): ReactElement |
   const assetInfo = useAssetInfo(txInfo)
   const [details, setDetails] = useState<Details | undefined>()
 
+  const makeTitle = (txDirection: string, amountWithSymbol: string) => (
+    <Text size="xl" as="span">
+      {txDirection === TransferDirection.INCOMING ? 'Received ' : 'Sent '}
```

Comment:
Very picky but I would suggest moving the space outside of the ternary:

```suggestion
      {txDirection === TransferDirection.INCOMING ? 'Received' : 'Sent'}{' '}
```

This also needs to depend on whether it is queued or not. It is not always the past tense.